applications/nrf5340_audio/README.rst

```diff
@@ -1268,6 +1270,10 @@ Legal notices for the nRF5340 Audio DK
 
       Nordic Semiconductor ASA customers using or selling these products for use in such applications do so at their own risk and agree to fully indemnify Nordic Semiconductor ASA for any damages resulting from such improper use or sale.
 
+   Radio frequency notice
```

Comment:
The indentations (in the .rst file) looks inconsistent to me, but it looks fine in the built doc.

Reply:
That's magic :D All is good formatting-wise.